README.md

```diff
@@ -52,98 +69,54 @@ $ docker run -p 6379:6379 redis
 
 Alternatively, you can install the appropriate Redis package for your OS.
 
-### Direnv
+#### Direnv
 
 We keep required config environment variables in `.envrc` which is managed by
 [direnv](https://direnv.net/). Any time you modify that file, you'll need to run
 `direnv allow`. You'll see an error message telling you so when cd-ing into the
 project's directory.
 
-## Usage
+You need to have a .envrc file created. Check our .envrc.example to see the minimum variables to define and their pourpose. Run:
 
-### Authentication
+```
+cp .envrc.example .envrc
+```
+Edit the variables on .envrc adding the real data.
 
-The API we expose for Woocommerce to send its webhook requests requires token
-authentication.
+#### Django
 
-To generate one you need to create a Django superuser and then to create a token
-for it. You can do so running the commands:
+Create a superuser to interact with django
 
 ```
-python manage.py createsuperuser --username pau --email pau@example.com
-python manage.py drf_create_token pau
+python manage.py createsuperuser --username woocommerce --email woocommerce@example.com
 ```
 
-This will give you the token to use with the `Authorization` HTTP header in your
-requests like `Authorization: Token 9944b09199c62bcf9418ad846dd0e4bbdfc6ee4b`.
-See:
-https://www.django-rest-framework.org/api-guide/authentication/#tokenauthentication.
-
-### Running in development
-
-We recommend [HTTPie](https://httpie.io/) to send requests to the API. It helps
-a great deal in managing the request bodies and authentication against the API.
-
-Once installed, make sure you have a user and a token for it in the DB.
-You can build an anonymous session like the one below.
-
-```json
-# session.json
-{
-    "__meta__": {
-        "about": "HTTPie session file",
-        "help": "https://httpie.org/doc#sessions",
-        "httpie": "1.0.3"
-    },
-    "auth": {
-        "password": null,
-        "type": null,
-        "username": null
-    },
-    "cookies": {},
-    "headers": {
-        "Accept": "application/json, */*",
-        "Authorization": "Token <put_your_token_here>"
-    }
-}
+
+Once all previous points are setup run migration to create the necessary tables on django
+
 ```
+python manage.py makemigration
```

Comment:
This step is not needed. This is the one that will create the migration files. All we need is to execute those that are already in the repo. If you run this, you won't see any result. If you do, it's because we forgot to create them.

Reply:
Fixed by https://github.com/coopdevs/lazona_connector/pull/92/commits/38ca9947aefcc0ee46ba1921c4d28a611855dda3